frontend/src/components/home/listcontent/SubjectItem.vue

```diff
@@ -4,7 +4,7 @@
             <h3>{{ subject.name }}</h3>
             <v-skeleton-loader v-if="isInstructorsLoading" type="text" />
             <p v-else-if="instructors!.length > 0" class="teacher">
-                {{ instructors![0].given_name }}
+                {{ instructors![0].given_name + " " + instructors![0].surname }}
```

Comment:
Is het een idee om deze eerst te sorteren op `is_teacher`, dan is de kans groter dat de naam van de effectieve prof hier te zien zal zijn, en niet die van een assistent.

Reply:
kheb dit aan issue #229 toegevoegd